js/pulse.js

```diff
@@ -916,15 +965,17 @@
                 }
             };
 
-            var _onAdFinished = function(){
+            var _onAdFinished = function() {
                 amc.notifyLinearAdEnded(1);
+                enableAdScreenPointerEvents();
             };
 
-            var _onAdSkipped = function(){
+            var _onAdSkipped = function() {
                 amc.notifyLinearAdEnded(1);
+                enableAdScreenPointerEvents();
```

Comment:
Is there a similar callback for when an ad errors out? If so, should we also enableAdScreenPointerEvents there as well?